### ISSUE TITLE
[Monster][Mulgore] Arra'chea

### ISSUE THREAD

**Issue by xnemeyethx** [dalaranwow/dalaran-wow#5662]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

There is another Arra'chea in the Mulgore.

**Expected behaviour**: Tell us what should happen instead

There should be only one such NPC. The one that does not move should be deleted.

**Steps to reproduce the problem**:

1. Go to Mulgore.
2. Find two Arra'cheas.
3. 

**Include proofs for this behaviour**

https://imgur.com/a/4Dh6M1t

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wotlk.evowow.com/?npc=3058
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
